backend/application/syson-application/src/test/resources/scripts/SysMLv2-SimpleProject.sql

```diff
@@ -0,0 +1,99 @@
+--
```

Comment:
I believe all new .sql files you've created should be move in a subfolder.
The "/scripts" folder should only contains subfolders and three files : cleanup.sql, dump-test-data.sh, and initialize-test-data.sh

Reply:
Done.